files/lib/gluon/ssid-changer/ssid-changer.sh

```diff
@@ -33,14 +33,14 @@ then
 	echo "Gateway TQ is $GATEWAY_TQ node is online"
 	for HOSTAPD in $(ls /var/run/hostapd-phy*); do # check status for all physical devices
 		CURRENT_SSID=`grep "^ssid=$ONLINE_SSID" $HOSTAPD | cut -d"=" -f2`
-		if [ $CURRENT_SSID == $ONLINE_SSID ]
+		if [ "X$CURRENT_SSID" == "X$ONLINE_SSID" ]
```

Comment:
`if [ "$CURRENT_SSID" == "$ONLINE_SSID" ]` should be sufficient for casting to string. There's no need to prepend an `X`.